packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.html

```diff
@@ -1,44 +1,32 @@
 <template>
-    <template if:true={isChecked}>
-      <li class="slds-var-m-vertical_small slds-grid">
-        <div class="slds-col facet__value-option slds-p-around_none">
-          <lightning-input 
-            type="checkbox"
-            variant="label-hidden"
-            label="Facet value option"
-            name="facet value checkbox"
-            onchange={onSelect}
-            checked
-          >
-          </lightning-input>
-          <span class="slds-var-m-right_xx-small slds-var-m-top_xxx-small slds-text-body_small nowrap facet__value-text facet__value_selected">{formattedFacetValue}</span>
-          <span class="nowrap slds-text-body_small facet__number-of-results facet__value_selected">({item.numberOfResults})</span>
-        </div>
-      </li>
-    </template>
-  
-    <template if:false={isChecked}>
-      <li class="slds-var-m-vertical_small slds-grid">
-        <div class="slds-col facet__value-option slds-p-around_none">
-          <lightning-input 
-            type="checkbox"
-            variant="label-hidden"
-            label="Facet value option"
-            name="facet value checkbox"
-            onchange={onSelect}
-          >
-          </lightning-input>
-          <template if:true={isStandardFacet}>
-            <span class="slds-var-m-right_xx-small slds-var-m-top_xxx-small slds-text-body_small nowrap facet__value-text facet__value_idle">
-              <lightning-formatted-rich-text class="slds-truncate" value={item.highlightedResult}></lightning-formatted-rich-text>
-            </span>
-          </template>
-          <template if:false={isStandardFacet}>
-            <span class="slds-var-m-right_xx-small slds-var-m-top_xxx-small slds-text-body_small nowrap facet__value-text facet__value_idle">{formattedFacetValue}</span>
-          </template>
-          <span class="nowrap slds-text-body_small facet__number-of-results">({numberOfResults})</span>
-        </div>
-      </li>
-    </template>
+  <li class="slds-grid">
+    <div onclick={onSelect} class="facet__value-container facet__value-option slds-size_1-of-1 slds-p-around_x-small">
+      <template if:false={displayAsLink}>
+        <lightning-input 
+          type="checkbox"
+          variant="label-hidden"
+          label="Facet value option"
+          name="facet value checkbox"
+          onchange={onSelect}
+          checked={isChecked}
```

Comment:
Wow okay nevermind me, they let you do it if it's coming from JavaScript!

Reply:
`<input type="checkbox" checked="false"></input>` will still generate a checked checkbox.

But if you do it programatically: `document.getElementById("checkbox").checked = false;` then it will not be checked. 

And it looks like doing `checked={isChecked}` in an LWC is equivalent to doing 

`document.getElementById("checkbox").checked = isChecked;` 
and not to doing `checked="true|false"`

Today I learned!

Reply:
woohoo! 🎉